src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss

```diff
@@ -175,15 +175,21 @@ limitations under the License.
     height: 24px;
 }
 
-.mx_GroupView_group {
+.mx_GroupView_editing {
     border-top: 1px solid $primary-hairline-color;
 }
 
-.mx_GroupView_group_disabled {
+.mx_GroupView_editing_disabled {
     opacity: 0.3;
     pointer-events: none;
 }
 
+.mx_GroupView_editing .mx_RoomDirectory_table {
+    cursor: initial;
+    pointer-events: none;
+    opacity: 0.7;
```

Comment:
rather than inventing a new form of fadable which is inconsistent with the existing one we use everywhere else on that page, please can we just hit it with `mx_fadable` and apply `mx_fadable_faded` when it's disabled?

Reply:
yep, I'd totally blanked that I'd done all that...